project/src/ui/hud/SmceHud.gd

```diff
@@ -87,6 +93,13 @@ func _set_vis(visible, node = null) -> void:
 	
 	tween.start()
 
+func _on_edit_btn() -> void:
+	get_focus_owner().release_focus()
+	if (code_editor_initialized==0):
+		get_tree().root.add_child(code_editor)
+		code_editor_initialized = 1
```

Comment:
this is quite bad, either do a null check and call instance then or just add the editor on ready